Jenkinsfile

```diff
@@ -1,216 +1,64 @@
 #!groovy
 
-@Library('github.com/mozmar/jenkins-pipeline@20170214.1')
+@Library('github.com/mozmar/jenkins-pipeline@20170315.1')
 
-def config
-def utils
-
-stage ('Checkout') {
-    node {
-        checkout scm
-        sh 'git submodule sync'
-        sh 'git submodule update --init --recursive'
-        // clean up
-        sh 'make clean'
-        // load the config
-        config = readYaml file: 'jenkins.yml'
-        // load the utility functions used below
-        utils = load 'jenkins/utils.groovy'
-        // defined in the Library loaded above
-        setGitEnvironmentVariables()
-        setConfigEnvironmentVariables(config)
-        // save the files for later
-        stash name: 'scripts', includes: 'bin/,docker/'
-        stash name: 'tests', includes: 'tests/,requirements/'
-        stash 'workspace'
+def loadBranch(String branch) {
+    // load the utility functions used below
+    utils = load 'jenkins/utils.groovy'
+    is_demo_branch = ( env.BRANCH_NAME ==~ /^demo\/[\w-]+$/ )
```

Comment:
I see that you've [documented this change](https://github.com/mozilla/bedrock/pull/4808/files#diff-b4f2733694997b4f1983468feb1dc9ffR140) and this is intentional, and I agree with the new prefix aesthetically, but I'd suggest modifying the regex slightly to accommodate the existing branches with the current `demo__` prefix, perhaps something like `/^demo[\/_\w-]+$/` (untested)